mantidimaging/gui/windows/welcome_screen/view.py

```diff
@@ -11,9 +10,9 @@
 class WelcomeScreenView(BaseDialogView):
     def __init__(self, parent, presenter):
         super().__init__(parent, "gui/ui/welcome_screen_dialog.ui")
-        base_path = finder.ROOT_PATH
 
-        bg_image = os.path.join(base_path, "gui/ui/images/welcome_screen_background.png")
+        # The background image URL must use forward slashes on both Windows and Linux
+        bg_image = finder.ROOT_PATH.replace('\\', '/') + '/gui/ui/images/welcome_screen_background.png'
```

Comment:
I think it would be better to switch `finder.ROOT_PATH` to be a Path (from pathlib) rather than a string. Then the separators should be correct automatically. Would looks something like:

`bg_image = finder.ROOT_PATH / "gui" / "ui" ...`

(will probably need changes in a few other places)

Reply:
Sorry, I think I'm getting a bit confused about the suggestion. `finder.ROOT_PATH` is outputting a Windows style path on Windows (i.e. using back slashes). The issue here was because the code uses the path to set the style `border-image:url({bg_image});` and it seemed that the url here needed to have forward slashes, regardless of platform.

In other circumstances where we're looking up a path (rather than using it to set a style sheet value) Windows can actually cope with backward, forward or a mix of slashes.

Let me know if I'm missing the point of the suggestion, though.